lib/src/devtools_integration/_registration.dart

```diff
@@ -51,15 +54,18 @@ bool setupDevToolsIntegration(
         UnexpectedRequestTypeError(request.runtimeType),
       ).toServiceResponse();
     } catch (error, stack) {
+      // ignore: avoid_print
       print(
```

Comment:
Nit: can we use `dart:io` here? If so, consider converting `print` to `stdout.writeln` and `stderr.writeln` where appropriate.

Reply:
Leak tracker will be used eventually for web. Made output a utility method.